sycl/source/detail/usm/usm_dispatch.cpp

```diff
@@ -347,19 +348,19 @@ void USMDispatcher::memAdvise(pi_queue Queue, const void *Ptr, size_t Length,
 
     if (mEmulated) {
       // memAdvise does nothing here
-      PI_CHECK(clEnqueueMarkerWithWaitList(
-          CLQueue, 0, nullptr, reinterpret_cast<cl_event *>(Event)));
+      PI_TRACE_OCL_CALL(clEnqueueMarkerWithWaitList,
```

Comment:
Please not add this macro (to not have people add more uses of it) and handle these cases manually as done above. Also please add TODO to rewrite USM handling with PI

Reply:
@smaslov-intel 
Please confirm if this is fine.